lib/src/user/userdata.dart

```diff
@@ -0,0 +1,30 @@
+// ignore: public_member_api_docs
+class UserData {
+  // ignore: public_member_api_docs
+  UserData(
+    this.id,
+    this.displayName,
+    this.email,
+    this.storageLocation,
+  );
+
+  // ignore: public_member_api_docs
+  factory UserData.fromJson(Map<String, dynamic> json) => UserData(
```

Comment:
Wouldn't an object mapper library like [dart_josn_mapper](https://pub.dev/packages/dart_json_mapper) come in handy for this?

Just a thought for the future.

Reply:
Sounds like a good idea.